src/index.html

```diff
@@ -1,22 +1,204 @@
+          <img src="images/Image-2.png" alt="BEOPLAY HX" class="recommended__item-img">
+          <h2 class="recommended__item-description">BEOPLAY HX</h2>
+          <p class="recommended__item-text">Wireless noise-cancelling headphones. Listen in comfort whether you're on a
+            call, working from home or taking a moment for yourself.</p>
+          <p class="recommended__item-price">499.00</p>
+        </article>
+
+        <article class="recommended__item">
+          <img src="images/Image-3.png" alt="BEOSOUND THEATRE" class="recommended__item-img">
+          <h2 class="recommended__item-description">BEOSOUND THEATRE</h2>
+          <p class="recommended__item-text">Four patent-pending acoustic innovations in one sleek soundbar envelop you
+            in sound from every angle. Made to fit any TV, and last for generations.</p>
+          <p class="recommended__item-price">3199.00</p>
+        </article>
+      </div>
+    </section>
+
+    <section class="categories">
+      <h1 class="section-title">Sound. Vision. It’s your pick.</h1>
+      <article class="categories__item">
+        <img src="images/Portable.png" alt="Portable speakers" class="categories__item-img">
+        <img src="images/Portable-2.png" alt="Portable speakers" class="categories__item-img">
+      </article>
+      <h3 class="categories__description">Portable speakers</h3>
+
+      <article class="categories__item">
+        <img src="images/Earphones.png" alt="Earphones" class="categories__item-img">
+        <img src="images/Earphones-2.png" alt="Earphones" class="categories__item-img">
+      </article>
+      <h3 class="categories__description">Earphones</h3>
+
+      <article class="categories__item">
+        <img src="images/SmartHome.png" alt="Smart Home" class="categories__item-img">
+        <img src="images/SmartHome-2.png" alt="Smart Home" class="categories__item-img">
+      </article>
+      <h3 class="categories__description">Smart Home</h3>
+
+      <button class="button button--view">View all</button>
+    </section>
+
+    <section class="timeless">
+      <div class="timeless__img"></div>
+      <div class="timeless__item">
+        <h1 class="timeless__item-title">Timeless, for 50 years and counting</h1>
+        <p class="timeless__item-text">Bang & Olufsen is expanding its Recreated Classics Program with the launch of
+          Beosystems – a limited edition music system that transcends time by bridging the gap between one of our iconic
+          designs from 1972 and today’s cutting-edge digital technology.</p>
+      </div>
+    </section>
+
+  </main>
+
+  <footer class="footer">
+
+    <section class="contacts" id="contacts">
+      <div class="container">
+        <h2 class="section-title">Contact us</h2>
+        <div class="contacts__content">
+          <form action="#" class="contacts__form" onsubmit="this.reset(); return false;">
+            <input type="text" name="name" class="contacts__input" placeholder="Name" />
```

Comment:
All inputs should be required [Reference](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md#:~:text=All%20form%20fields%20should%20be%20required%20and%20have%20correct%20types%20(email%2C%20tel%2C%20etc.))